civihr_default_theme/scss/partials/_menu.scss

```diff
@@ -69,8 +69,11 @@
 }
 
 .chr_header__home-menu {
+  color: $light;
```

Comment:
Previously the menu was always a `span`, now sometimes its `anchor` when `civicrm` access is not there. So had to explicitly mention the color. Result is same as before.